### ISSUE TITLE
Cobbler import doesnt work with relative paths

### ISSUE THREAD

**Issue by jmaas** [cobbler/cobbler#1613]
```
[root@centos-7-amd64 data]# cobbler import --name=sles-12.1-x86_64 --path=./iso
task started: 2016-01-22_140455_import
task started (id=Media import, time=Fri Jan 22 14:04:55 2016)
Exception occured: <class 'cobbler.cexceptions.CX'>
Exception value: 'Command failed'
Exception Info:
  File "/usr/lib/python2.7/site-packages/cobbler/remote.py", line 82, in run
    rc = self._run(self)
   File "/usr/lib/python2.7/site-packages/cobbler/remote.py", line 226, in runner
    self.logger
   File "/usr/lib/python2.7/site-packages/cobbler/api.py", line 876, in import_tree
    utils.run_this(rsync_cmd, (spacer, mirror_url, path), self.logger)
   File "/usr/lib/python2.7/site-packages/cobbler/utils.py", line 889, in run_this
    die(logger,"Command failed")
   File "/usr/lib/python2.7/site-packages/cobbler/utils.py", line 114, in die
    raise CX(msg)

!!! TASK FAILED !!!
```

**Comment by ixs** [comment]
This change actually breaks the import via rsync workflow.

> Mon Dec 26 06:24:16 2016 - INFO | REMOTE start_task(Media import); event_id(2016-12-26_062416_import); user(?)
> Mon Dec 26 06:24:17 2016 - INFO | import_tree; ['/var/www/cobbler/ks_mirror/rsync:/ftp.us.debian.org/debian/dists/jessie/main/installer-amd64/current/images/netboot/debian-installer/amd64', 'jessie-net', None, None, '--quiet']
> Mon Dec 26 06:24:17 2016 - INFO | importing from a network location, running rsync to fetch the files first
> Mon Dec 26 06:24:17 2016 - INFO | running: rsync -a  '/var/www/cobbler/ks_mirror/rsync:/ftp.us.debian.org/debian/dists/jessie/main/installer-amd64/current/images/netboot/debian-installer/amd64/' /var/www/cobbler/ks_mirror/jessie-net-amd64 --progress --quiet
> Mon Dec 26 06:24:17 2016 - INFO | received on stdout:
> Mon Dec 26 06:24:17 2016 - DEBUG | received on stderr: rsync: change_dir "/var/www/cobbler/ks_mirror/rsync:/ftp.us.debian.org/debian/dists/jessie/main/installer-amd64/current/images/netboot/debian-installer/amd64" failed: No such file or directory (2)
> rsync error: some files/attrs were not transferred (see previous errors) (code 23) at main.c(1052) [sender=3.0.9]
> 
> Mon Dec 26 06:24:17 2016 - INFO | Exception occured: <class 'cobbler.cexceptions.CX'>
> Mon Dec 26 06:24:17 2016 - INFO | Exception value: 'Command failed'
> Mon Dec 26 06:24:17 2016 - INFO | Exception Info:
>   File "/usr/lib/python2.7/site-packages/cobbler/utils.py", line 106, in die
>     raise CX(msg)

A possible fix would look like this:
```
--- /tmp/cli.py	2016-12-26 06:32:27.802173616 -0500
+++ /usr/lib/python2.7/site-packages/cobbler/cli.py	2016-12-26 06:27:27.725191502 -0500
@@ -514,7 +514,7 @@
             self.parser.add_option("--kickstart",    dest="kickstart_file", help="assign this kickstart file")
             self.parser.add_option("--rsync-flags",  dest="rsync_flags",    help="pass additional flags to rsync")
             (options, args) = self.parser.parse_args()
-            if options.path:
+            if options.path and not options.path.startswith("rsync://"):
                 # convert relative path to absolute path
                 options.path = os.path.abspath(options.path)
             task_id = self.start_task("import",options)
```

I am not sure though, if this is fixing _all_ potential problems that might have been introduced by this change.

**Comment by jmaas** [comment]
Was fixed in master already, suggested fix has been applies to release28 branch.
So this will be fixed in the 2.8.5 release.
